.circleci/config.yml

```diff
@@ -384,16 +419,36 @@ jobs:
       - wait-for-infrastructure
       - run:
           name: Running test section against a remote target
-          command: ./packages/fxa-content-server/scripts/test-ci-remote.sh
+          command: ./packages/fxa-content-server/scripts/test-ci-remote.sh 3
       - store_artifacts:
           path: ~/screenshots
           destination: screenshots
       - store_test_results:
           path: artifacts/tests
 
+  # These jobs are manually triggered for now. see .circleci/README.md
```

Comment:
@jbuck You can now run these three parts in parallel instead of `test-content-server-remote`